packages/cascara/src/modules/DataNumber/DataNumber.js

```diff
@@ -29,25 +30,34 @@ const DataNumber = ({
 }) => {
   const { isEditing, formMethods } = useContext(ModuleContext);
 
+  const conditionalLabelProps = getConditionalLabelProps(label, isLabeled);
+
   const renderEditing = (
-    <label htmlFor={label}>
-      {label && isLabeled && <span className={styles.LabelText}>{label}</span>}
+    <>
+      {label && isLabeled && (
+        <label htmlFor={attribute || label}>
+          <span className={styles.LabelText}>{label || attribute}</span>
+        </label>
+      )}
       <Input
+        {...conditionalLabelProps}
         {...rest}
         className={styles.Input}
         defaultValue={value}
-        id={label}
+        id={attribute || label}
         name={attribute || label}
         ref={formMethods?.register}
         type='number'
       />
-    </label>
+    </>
```

Comment:
label tag to wrap input

Reply:
This module is now wrapped by the new `LabeledModule` 👍